target/linux/mediatek/filogic/base-files/etc/board.d/02_network

```diff
@@ -181,6 +184,11 @@ mediatek_setup_macs()
 		wan_mac=$(mtd_get_mac_ascii Bdata ethaddr_wan)
 		label_mac=$wan_mac
 		;;
+	iptime,ax3000m)
+		label_mac=$(mtd_get_mac_binary "Factory" 0x4)
+		wan_mac=$(macaddr_add "$label_mac" 1)
+		lan_mac=$(macaddr_add "$label_mac" 3)
+		;;
```

Comment:
These are already assigned by devicetree and can be dropped.

Reply:
mac addresses cannot be changed without of these. lan,wan addresses are set to label_mac.

i moved config to lib/preinit/10_fix_eth_mac.sh for make it work.